src/js/components/FollowOrIgnore.jsx

```diff
@@ -1,79 +1,26 @@
 "use strict";
-
 import React, { Component, PropTypes } from "react";
 import { Button, ButtonToolbar } from "react-bootstrap";
-import VoterGuideStore from "../stores/VoterGuideStore";
+import GuideActions from "../actions/GuideActions";
 
 export default class FollowOrIgnore extends Component {
     static propTypes = {
-        organization_we_vote_id: PropTypes.string,
-        action: PropTypes.object.isRequired,
-        action_text: PropTypes.string,
-        OrganizationFollowed: PropTypes.string
+        organization_we_vote_id: PropTypes.string.isRequired,
     };
 
-    constructor(props) {
-        super(props);
-
-        this.state = {
-          OrganizationFollowed: this.props.OrganizationFollowed,
-        };
-    }
-
-  toggleFollow (evt) {
-    evt.stopPropagation();
-    if (this.state.OrganizationFollowed == "Yes")
-      this.props.action.stopFollowingOrg(this.props.organization_we_vote_id);
-    else
-      this.props.action.followOrg(this.props.organization_we_vote_id);
-  }
-
-  ignoreOrgLocal (evt) {
-    evt.stopPropagation();
-    this.props.action.ignoreOrg(this.props.organization_we_vote_id);
-  }
+    render () {
+      var ignoreFunc = GuideActions.ignore.bind(this, this.props.organization_we_vote_id);
```

Comment:
I will only add this one more time so I don't have to do it everywhere ... I think that `let` should be used in place of `var`